components/Layout/TwoCol.tsx

```diff
@@ -41,7 +41,7 @@ export const TwoCol = ({
         {
           "lg:*:w-1/2": !isSidebar,
           "lg:*:even:max-w-2/3 lg:*:odd:max-w-1/3": isSidebar,
-          "xl:max-w-900 2xl:max-w-[1300px]": isNarrow,
```

Comment:
I'm seeing a mix of width and height using `px` and `rem` units in this repo - please stick to rem unless it requires px for specific reasons 😄